tests/foreman/cli/test_discoveredhost.py

```diff
@@ -758,20 +459,6 @@ def test_positive_auto_provision_pxeless_host(self):
         :CaseImportance: Critical
         """
 
-    @pytest.mark.stubbed
-    @pytest.mark.tier3
-    def test_positive_list_discovered_host(self):
```

Comment:
we already do this as a part of
`test_positive_provision_pxeless_bios_syslinux`